### ISSUE TITLE
Process feedback after Current progress presentation

### ISSUE THREAD

**Issue by DianaDI** [DianaDI/compressedCV#34]
* correlation graph on number of labelled data, size of the object and accuracy  **[in progress]**
* on what data models were pre-trained? include to the presentation
* 600x600 (first priority) **[in progress]**
* relative difference on AP on detected objects - plot **[done]**
* show picture examples for each quality
* train longer - future work
* finetuning at least one model
* well explanation of graphs 
* keep one fastercnn - Faster rcnn inception v2 coco
* other compression techniques - for future work
* report 
* put the final presentation on github

**Comment by DianaDI** [comment]
* don't compare losses (different losses for different models/different pre-tained data)
